backend/src/tasks/sample_data/vulnerabilities.json

```diff
@@ -865,5 +865,71 @@
         "source": "cpe2cve",
         "notes": "",
         "actions": []
+    },
+    {
+        "title": "Exposed Emails",
+        "cve": null,
+        "cwe": null,
+        "cpe": null,
+        "description": "Emails associated with this Domain have been exposed in a breach. See Data field for a list of emails",
+        "cvss": null,
+        "severity": null,
+        "needsPopulation": false,
+        "state": "open",
+        "substate": "unconfirmed",
+        "source": "hibp",
+        "structuredData": { "emails": {
+            "Email_11":["breach12","breach17"],
```

Comment:
Can you make this sample data look more like real emails (so we can do something like `email_11@crossfeed.local`)